docs/webhook.md

```diff
@@ -0,0 +1,57 @@
+# Webhook
+
+Fleet by default using polling(15 seconds) to pull from git repo, but can be configured to use webhook instead. Fleet currently supports Github,
```

Comment:
nit:
```
By default, Fleet utilizes polling (default: 15 seconds) to pull from a Git repo.
However, this can be configured to utilize a webhook instead.
```

Reply:
:+1: fixed